### ISSUE TITLE
Set required six version in setup file

### ISSUE THREAD

**Issue by cmft** [tango-controls/pytango#269]
PyTango uses at least `six.ensure_binary` method  that is not able in all six version (since it was added in 1.12.0). The necessary six version should be specified e.g. in the setup file.

```
  File "/usr/lib/python2.7/dist-packages/tango/db.py", line 205, in __Database__generic_get_property
    e = six.ensure_binary(e, 'latin-1')

  reason = PyDs_PythonError
severity = ERR]
```

**Comment by ajoubertza** [comment]
Thanks @cmft - I will update the setup file.

**Comment by ajoubertza** [comment]
Fixed in #270.
